zuul.yaml

```
@@ -40,7 +41,15 @@
     name: auth-tox-integration
     description: Run auth integration tests
     parent: wazo-tox-integration-py39
-    pre-run: zuul.d/integration-pre-run.yaml    
+    timeout: 10800
+    vars:
+      integration_test_timeout: 60
+
+- job:
+    name: auth-tox-functional
+    description: Run auth functional tests
+    parent: wazo-tox-integration-py39
+    pre-run: zuul.d/functional-pre-run.yaml
```

You also need to tell Zuul to use the new `tox` environment:
```suggestion
    pre-run: zuul.d/functional-pre-run.yaml
    tox_envlist: functional
```